source/tests/system/nidcpower_driver_api_tests.cpp

```diff
@@ -102,6 +102,76 @@ class NiDCPowerDriverApiTest : public ::testing::Test {
     return response.error_message();
   }
 
+  ViBoolean get_bool_attribute(const char* channel_list, dcpower::NiDCPowerAttributes attribute_id)
+  {
+    ::grpc::ClientContext context;
+    dcpower::GetAttributeViBooleanRequest request;
+    request.mutable_vi()->set_id(GetSessionId());
+    request.set_channel_name(channel_list);
+    request.set_attribute_id(attribute_id);
+    dcpower::GetAttributeViBooleanResponse response;
+    ::grpc::Status status = GetStub()->GetAttributeViBoolean(&context, request, &response);
+    EXPECT_TRUE(status.ok());
+    expect_api_success(response.status());
+    return response.attribute_value();
+  }
+
+  ViInt32 get_int32_attribute(const char* channel_list, dcpower::NiDCPowerAttributes attribute_id)
+  {
+    ::grpc::ClientContext context;
+    dcpower::GetAttributeViInt32Request request;
+    request.mutable_vi()->set_id(GetSessionId());
+    request.set_channel_name(channel_list);
+    request.set_attribute_id(attribute_id);
+    dcpower::GetAttributeViInt32Response response;
+    ::grpc::Status status = GetStub()->GetAttributeViInt32(&context, request, &response);
+    EXPECT_TRUE(status.ok());
+    expect_api_success(response.status());
+    return response.attribute_value();
+  }
+
+  ViInt64 get_int64_attribute(const char* channel_list, dcpower::NiDCPowerAttributes attribute_id)
+  {
+    ::grpc::ClientContext context;
+    dcpower::GetAttributeViInt64Request request;
+    request.mutable_vi()->set_id(GetSessionId());
+    request.set_channel_name(channel_list);
+    request.set_attribute_id(attribute_id);
+    dcpower::GetAttributeViInt64Response response;
+    ::grpc::Status status = GetStub()->GetAttributeViInt64(&context, request, &response);
+    EXPECT_TRUE(status.ok());
+    expect_api_success(response.status());
+    return response.attribute_value();
+  }
+
+  ViReal64 get_real64_attribute(const char* channel_list, dcpower::NiDCPowerAttributes attribute_id)
+  {
+    ::grpc::ClientContext context;
+    dcpower::GetAttributeViReal64Request request;
+    request.mutable_vi()->set_id(GetSessionId());
+    request.set_channel_name(channel_list);
+    request.set_attribute_id(attribute_id);
+    dcpower::GetAttributeViReal64Response response;
+    ::grpc::Status status = GetStub()->GetAttributeViReal64(&context, request, &response);
+    EXPECT_TRUE(status.ok());
+    expect_api_success(response.status());
+    return response.attribute_value();
+  }
+
+  std::string get_string_attribute(const char* channel_list, dcpower::NiDCPowerAttributes attribute_id)
+  {
+    ::grpc::ClientContext context;
+    dcpower::GetAttributeViStringRequest request;
+    request.mutable_vi()->set_id(GetSessionId());
+    request.set_channel_name(channel_list);
+    request.set_attribute_id(attribute_id);
+    dcpower::GetAttributeViStringResponse response;
+    ::grpc::Status status = GetStub()->GetAttributeViString(&context, request, &response);
```

Comment:
Can you add new line before calling actual API to make it more readable?

Reply:
done.